.github/workflows/test.yaml

```diff
@@ -168,37 +162,20 @@ jobs:
             trial-args: '--reactor=select'
 
           # Windows, newest Python supported version with iocp reactor.
-          - python-version: '3.12'
+          - python-version: '3.13'
             runs-on: 'windows-2022'
             tox-env: 'alldeps-withcov-windows'
             job-name: 'win-default-tests-iocp'
             # Distributed trial is not yet suported on Windows.
             trial-args: '--reactor=iocp'
 
-          # Prerelease Python 3.13 on Windows with select reactor.
-          # pywin32 doesn't have wheels for Python 3.13 yet.
-          # See https://github.com/mhammond/pywin32/issues/2294
-          # # Prerelease Python 3.13 on Windows with select reactor.
-          # - python-version: '3.13'
-          #   runs-on: 'windows-2022'
-          #   tox-env: 'alldeps-withcov-windows'
-          #   job-name: 'win-3.13-tests-select'
-          #   # Distributed trial is not yet suported on Windows so we overwrite
-          #   # the default trial-args to remove concurrent runs and to
-          #   # select a reactor.
-          #   trial-args: '--reactor=select'
-
-          # Prerelease Python 3.13 on Windows with iocp reactor.
-          # pywin32 doesn't have wheels for Python 3.13 yet.
-          # See https://github.com/mhammond/pywin32/issues/2294
-          # - python-version: '3.13'
-          #   runs-on: 'windows-2022'
-          #   tox-env: 'alldeps-withcov-windows'
-          #   job-name: 'win-3.13-tests-iocp'
-          #   # Distributed trial is not yet suported on Windows so we overwrite
-          #   # the default trial-args to remove concurrent runs and to
-          #   # select a reactor.
-          #   trial-args: '--reactor=iocp'
+          # Windows, newest Python supported version with select reactor.
```

Comment:
I am not sure if we need this.

The goal was to reduce the test matrix and I was hoping that `# Windows, minimum Python version with select reactor.` can provide good coverage for the select reactor.

-------

We have dedicate reactor tests that are executed with each run, regardless of the default reactor used by `trial`

-----

We can keep this extra build if you think it's usefull.